src/components/MenuButtons.vue

```diff
@@ -51,20 +54,38 @@ function clearStructureSearch(event: Event){
                 @click="openDropdownOpen = true"
                 @keypress.enter="openDropdownOpen = true"
             />
-            <font-awesome-icon
-                icon="fa-magnifying-glass"
-                class="button"
-                tabindex="0"
-                :class="{
-                    open: searchBiomeDropdownOpen,
-                    active: searchStore.biomes.size > 0
-                }"
-                :title="$t('menu.search_biome.title')"
-                @click="searchBiomeDropdownOpen = true"
-                @keypress.enter="searchBiomeDropdownOpen = true"
-                @contextmenu="clearBiomeSearch"    
-                @dblclick="clearBiomeSearch"
-            />
+            <span class="group-button" :class="{active: !searchStore.disabled}">
+                <font-awesome-icon
+                    icon="fa-magnifying-glass"
+                    class="button"
+                    tabindex="0"
+                    :class="{
+                        open: searchBiomeDropdownOpen,
+                        active: searchStore.biomes.size > 0
+                    }"
+                    :title="$t('menu.search_biome.title')"
+                    @click="searchBiomeDropdownOpen = true"
+                    @keypress.enter="searchBiomeDropdownOpen = true"
+                    @contextmenu.prevent="clearBiomeSearch"    
+                    @dblclick.prevent="clearBiomeSearch"
+                />
+                <font-awesome-icon
+                    v-show="!searchStore.disabled"
+                    icon="fa-toggle-on"
```

Comment:
This shouldn't be 2 separate elements, rather just use `:icon="searchStore.disabled ? 'fa-toggle-off' : 'fa-toggle-on'`